chapters/packages.tex

```diff
@@ -223,6 +223,7 @@ \subsection{Directory Hierarchy Mapping}\label{mapping-a-package-class-hierarchy
 In order to preserve the order of sub-entities it is advisable to create a file \filename{package.order} where each line contains the name of one class or constant (using its Modelica \lstinline!IDENT! form).
 If a \filename{package.order} is present when reading a structured entity the classes and constants are added in this order; if the contents does not exactly match the classes and constants in the package, the resulting order is tool specific and a warning may be given.
 Classes and constants that are stored in \filename{package.mo} are also present in \filename{package.order} but their relative order should be identical to the one in \filename{package.mo} (this ensures that the relative order between classes and constants stored in different ways is preserved).
+Any extends-clause (stored in \filename{package.mo}) may be present in \filename{package.order} in the form \lstinline!extends A.B! (one space, no semicolon), but should be omitted if the extends-clause(s) precede normal classes.
```

Comment:
The main form is the singular:
```suggestion
Any extends-clause (stored in \filename{package.mo}) may be present in \filename{package.order} in the form \lstinline!extends A.B! (one space, no semicolon), but should be omitted if the extends-clause(s) precedes normal classes.
```

Reply:
More complete suggestion addressing additional concerns in review comment:
```suggestion
A \lstinline!extends!-clause present in \filename{package.mo} may be listed in \filename{package.order} in the form \lstinline!extends A.B! (one space, no semicolon), except at the start of \filename{package.order}.
That is, the \filename{package.order} must contain at least one entry without \lstinline!extends! before the first \lstinline!extends!-clause entry.
\begin{nonnormative}
The rule regarding entries at the start of \filename{package.order} offers full backwards compatibility with older packages in the common case of all \lstinline!extends!-clauses appearing first in \filename{package.mo}.
\end{nonnormative}
```

Reply:
The suggestion of not allowing at the start doesn't fully work, since there can also be constants in package.order. 

Since they cannot be stored in separate files they must be in package.mo - and even if usually the extends-clause precede all of them it is possible to have a mix, but as long as the extends-clause precedes any _class_ there is no need to include the extends in package.order.

Reply:
I think that this is better addressed by my proposal below in https://github.com/modelica/ModelicaSpecification/pull/3635#issuecomment-2575008137.  According to that proposal, the problematic packages would just end falling into the deprecated behavior.